invenio_search/api.py

```diff
@@ -65,8 +66,13 @@ def search(self, user_info=None, collection=None, **kwargs):
 
         for walker in search_walkers():
             query = query.accept(walker)
-        return Results(query)
 
+        index = cfg["SEARCH_ELASTIC_COLLECTION_INDEX_MAPPING"].get(
```

Comment:
we should create index alias for all of them or search in all indexes.

`records-hep`, `records-jobs` -> `records-*`